### PR TITLE
Add target framework for .Net Core 3.1

### LINKED ISSUE #78: Migrate to .Net Core 3.1
Hi there!

First of all thanks for the great library!
Is there something blocking you from migrating to .Net Core 3.1?

### PR DESCRIPTION
**Pull request by punker76**
### Description of Change ###

This PR adds the target framework for .Net Core 3.1.

### Bugs Fixed ###

Closes #78 

### API Changes ###

Added:
 - netcoreapp3.1 target framework

### Behavioral Changes ###

-/-

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Review by brianlagunas — APPROVED** [review]

File: src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj [hunk 1]
```diff
@@ -103,7 +103,7 @@
   <Target Name="IncludeProjectToProjectAssets">
     <ItemGroup>
       <BuildOutputInPackage Include="$(ProjectDir)bin\$(Configuration)\net45\Design\Microsoft.Xaml.Behaviors.DesignTools.dll"
-                            Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+                            Condition="'$(TargetFramework.StartsWith('netcoreapp'))">
```
Comment (pedrolamas) [inline src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj:106]: Frankly I'd prefer to test both frameworks or at least test for "netcoreapp3." prefix.

**Review by pedrolamas — CHANGES_REQUESTED** [review]

File: src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj (hunk 1 above)
Comment (punker76) [reply src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj]: @pedrolamas I changed it to `netcoreapp3.`

**Review by pedrolamas — APPROVED** [review]

**Comment by brianlagunas** [comment]
Hey @pedrolamas and @DVaughan, any idea why the build hasn't reported back? We can't merge until that process has been reported completed.

**Comment by pedrolamas** [comment]
No idea, I was thinking the same myself...

**Comment by DVaughan** [comment]
It appears that the pipeline was not triggered. I don't know why not. Perhaps make an update and see if it is triggered.

**Comment by pedrolamas** [comment]
@punker76 can you please force-push (or make any other change) to your branch so we can check if the build pipeline starts as expected? Thank you!

**Comment by punker76** [comment]
@pedrolamas @DVaughan @brianlagunas I pushed a minor change, but it seems like the build pipeline doesn't like this PR. I can close this and try to create a new branch and PR...

**Comment by brianlagunas** [comment]
@DVaughan @pedrolamas Looks like we have an issue with the build server. Who can look into that?

**Comment by DVaughan** [comment]
Currently looking into it.

**Comment by DVaughan** [comment]
@punker76 would you please try that again. I've switched to a GitHub action and it's waiting for status. But, I'm hoping it will trigger the action with another commit.

**Comment by DVaughan** [comment]
I've verified the build locally for this PR. I think the PR might need to be recreated for the action to trigger, so I'm merging without the status.

**Pull request merged**